src/transaction/components/TransactionButton.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { TransactionProvider } from './TransactionProvider';
+import { WagmiProvider, createConfig, http } from 'wagmi';
+import { baseSepolia } from 'viem/chains';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { TransactionButton } from './TransactionButton';
+
+const wagmiConfig = createConfig({
+  chains: [baseSepolia],
+  transports: {
+    [baseSepolia.id]: http(),
+  },
+});
+
```

Comment:
Out of curiosity, were you able to get this Story working, because I was trying build some complex story with different Wagmi providers, and they were not quite working.

Reply:
i was able to get it working for `TransactionButton` but i tried to create a more complex one for the `Transaction` component and haven't been able to figure that one out yet which is why i created the docs pages

Reply:
All good, I asked Sid to help us out with Storybook in the future.